translations/de_CH/atrium-translations-de_CH-jvm/src/main/kotlin/ch/tutteli/atrium/translations/DescriptionOptionalAssertion.kt

```diff
@@ -0,0 +1,8 @@
+package ch.tutteli.atrium.translations
+
+import ch.tutteli.atrium.reporting.translating.StringBasedTranslatable
+
+enum class DescriptionOptionalAssertion(override val value: String) : StringBasedTranslatable {
+    EMPTY("leer"),
+    NOT_EMPTY("nicht leer"),
```

Comment:
please remove NOT_EMPTY will not be used

Reply:
Removed